src/TestSuite/Migrator.php

```diff
@@ -160,7 +160,11 @@ protected function shouldDropTables(Migrations $migrations, array $options): boo
 
         foreach ($migrations->status($options) as $migration) {
             if ($migration['status'] === 'up') {
-                Log::write('debug', 'One or more additional migrations detected.');
+                Log::write(
+                    'debug',
+                    'One or more migrations need to be applied. ' .
+                    "Migration with source={$migration['name']} id={$migration['id']} needs to be applied."
```

Comment:
Do we have any way to format a list?

Reply:
Not really. We also break out of the loop early. We could instead collect all the output and format a list.

Reply:
Can we at least line break and prefix with something? A `-` or a space or whatever?